File: SINAN.Domain/Entities/Sinan/Sinan.NotificacaoIndividual.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SINAN.Domain.Entities
{
    public class Sinan_NotificacaoIndividual
    {
        public virtual int id { get; set; }
        public virtual string nome_paciente { get; set; }
        public virtual DateTime? data_nascimento { get; set; }
        public virtual string idade { get; set; }
        public virtual int? ou_idade { get; set; }
        public virtual string sexo { get; set; }
        public virtual int? gestante { get; set; }
        public virtual int? raca { get; set; }
        public virtual int? escolaridade { get; set; }
        public virtual int? continua_estudando { get; set; }
        public virtual int? ano_de_escolaridade { get; set; }
        public virtual int? rede_ensino { get; set; }
        public virtual int? esfera_ensino { get; set; }
        public virtual string nome_escola { get; set; }
        public virtual string num_cartaosus { get; set; }
        public virtual string num_cartaonis { get; set; }
        public virtual string nome_mae { get; set; }
        public virtual int? tem_nome_pai_registro { get; set; }
    }
}

File: SINAN/Scripts/sinan/Sinan.NotificacaoIndividual.js
$(document).ready(function () {

    // Eventos

    $("#sexo").change(function () {

        var sexo = $("#sexo");

        if (sexo.val() == 70) {

            $("#gestante").prop('disabled', false);

        } else {

            $("#gestante").val(null);
            $("#gestante").prop('disabled', true);
        }
    });

    $("#continua_estudando").change(function () {

        var continua_estudando = $("#continua_estudando");

        if (continua_estudando.val() == 1) {

            $("#ano_de_escolaridade").prop('disabled', false);
            $("#rede_ensino").prop('disabled', false);
            $("#nome_escola").prop('disabled', false);

        } else {

            $("#ano_de_escolaridade").val(null);
            $("#ano_de_escolaridade").prop('disabled', true);

            $("#rede_ensino").val(null);
            $("#rede_ensino").prop('disabled', true);

            $("#esfera_ensino").val(null);
            $("#esfera_ensino").prop('disabled', true);

            $("#nome_escola").val('');
            $("#nome_escola").prop('disabled', true);
        }
    });

    $("#rede_ensino").change(function () {

        var rede_ensino = $("#rede_ensino");

        if (rede_ensino.val() == 1) {

            $("#esfera_ensino").prop('disabled', false);

        } else {

            $("#esfera_ensino").val(null);
            $("#esfera_ensino").prop('disabled', true);
        }
    });

});